packages/extra/etcd/templates/datastore.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: kamaji.clastix.io/v1alpha1
```

Comment:
I'm not sure here, can you clarify?

Reply:
This file is from another pull request https://github.com/aenix-io/cozystack/pull/95 and will be updated to this version once the first one is merged